lib/model/model_test.go

```diff
@@ -3267,12 +3268,7 @@ func TestSanitizePath(t *testing.T) {
 func TestConnCloseOnRestart(t *testing.T) {
 	w, fcfg := tmpDefaultWrapper()
 	m := setupModel(w)
-	defer func() {
-		m.Stop()
-		m.db.Close()
-		os.RemoveAll(fcfg.Filesystem().URI())
-		os.Remove(w.ConfigPath())
-	}()
+	defer cleanupModelAndRemoveDir(m, fcfg.Filesystem().URI())
```

Comment:
Not relevant to this, but we could just get all the folders from the config (that exits on the mode) and remove them, without having to pass a URI.

Reply:
Yes, in some cases. But some of the tests are running on `./testdata` which is committed to git, so there it shouldn't remove anything.